CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 ### Added
 
 ### Changed
+- Only support an operator nesting level up to 69 (#1113)
```

Comment:
Internally in libxml it's a nesting level limit, but when translated back to what appears in the selector, the limit 69 is the number of atomic expressions in a selector (i.e. without combinators), so I think the description should reflect that.

Reply:
Updated accordingly, rebased and repushed.